content/ja/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -0,0 +1,414 @@
+この[スクリプト](/examples/admin/resource/memory-available.sh)または[cgroupv2スクリプト](/examples/admin/resource/memory-available-cgroupv2.sh)は、kubeletが`memory.available`を計算するために実行する一連の手順を再現します。
+kubeletは、圧迫下でもメモリが再利用可能であると想定しているため、inactive_file(非アクティブなLRUリスト上のファイルベースのメモリのバイト数)を計算から除外します。
+
+On Windows nodes, the value for `memory.available` is derived from the node's global
+memory commit levels (queried through the [`GetPerformanceInfo()`](https://learn.microsoft.com/windows/win32/api/psapi/nf-psapi-getperformanceinfo)
+system call) by subtracting the node's global [`CommitTotal`](https://learn.microsoft.com/windows/win32/api/psapi/ns-psapi-performance_information) from the node's [`CommitLimit`](https://learn.microsoft.com/windows/win32/api/psapi/ns-psapi-performance_information). Please note that `CommitLimit` can change if the node's page-file size changes!
+
+#### ファイルシステムシグナル
+
+kubeletは、退避シグナル(`<identifier>.inodesFree`や`<identifier>.available`)で使用できる3つの特定のファイルシステム識別子を認識します:
+
+1. `nodefs`: ノードのファイルシステムであり、ローカルディスクボリューム、メモリにバックアップされていないemptyDirボリューム、ログストレージ、エフェメラルストレージなどに使用されます。
+   例えば、`nodefs`には`/var/lib/kubelet`が含まれます。
+
+1. `imagefs`: コンテナらランタイムがコンテナイメージ(読み取り専用レイヤー)とコンテナの書き込みレイヤーを格納するために使用できるオプションのファイルシステムです。
+
+1. `containerfs`: コンテナランタイムが書き込み可能なレイヤーを格納するために使用できるオプションのファイルシステムです。
+   メインファイルシステム(`nodefs`を参照)と同様に、ローカルディスクボリューム、メモリにバックアップされていないemptyDirボリューム、ログストレージ、エフェメラルストレージに使用されますが、コンテナイメージは含まれません。
+   `containerfs`を使用すると、`imagefs`ファイルシステムをコンテナイメージ(読み取り専用レイヤー)のみを格納するように分割できます。
+
+したがって、kubeletは通常コンテナファイルシステムについて次の3つのオプションを許可します:
+
+- すべてが単一の`nodefs`にある場合、「rootfs」または単に「root」として参照され、専用のイメージファイルシステムはありません。
+
+- コンテナストレージ(`nodefs`を参照)は専用のディスクにあり、`imagefs`(書き込み可能レイヤーと読み取り専用レイヤー)はルートファイルシステムから分離されています。
+  これはしばしば「分割ディスク」(または「分離ディスク」)ファイルシステムと呼ばれます。
+
+- コンテナファイルシステム`containerfs`(書き込み可能レイヤーを含む`nodefs`と同じ)がルートにあり、コンテナイメージ(読み取り専用レイヤー)は分離された`imagefs`に格納されています。
+  これはしばしば「分割イメージ」ファイルシステムと呼ばれます。
+
+kubeletは、これらのファイルシステムを現在の構成に基づいてコンテナランタイムから直接自動検出しようとし、他のローカルノードファイルシステムを無視します。
+
+kubeletは、他のコンテナファイルシステムやストレージ構成をサポートせず、現在イメージとコンテナに対して複数のファイルシステムをサポートしていません。
+
+### 非推奨のkubeletガベージコレクション機能
+
+一部のkubeletガベージコレクション機能は、退避に置き換えられるため非推奨となりました:
+
+| 既存フラグ | 理由 |
+| ------------- | --------- |
+| `--maximum-dead-containers` | 古いログがコンテナのコンテキスト外に保存されると非推奨になります |
+| `--maximum-dead-containers-per-container` | 古いログがコンテナのコンテキスト外に保存されると非推奨になります |
+| `--minimum-container-ttl-duration` | 古いログがコンテナのコンテキスト外に保存されると非推奨になります |
+
+### 退避閾値
+
+kubeletは、退避の判断を行うときに使用するカスタムの退避閾値を指定できます。
+[ソフト](#soft-eviction-thresholds)と[ハード](#hard-eviction-thresholds)の退避閾値を構成できます。
+
+退避閾値は`[eviction-signal][operator][quantity]`の形式を取ります:
+
+- `eviction-signal`は、使用する[退避シグナル](#eviction-signals)です。
+- `operator`は、`<`(より小さい)などの[関係演算子]((https://ja.wikipedia.org/wiki/%E9%96%A2%E4%BF%82%E6%BC%94%E7%AE%97%E5%AD%90#%E6%A8%99%E6%BA%96%E7%9A%84%E3%81%AA%E9%96%A2%E4%BF%82%E6%BC%94%E7%AE%97%E5%AD%90))です。
```

Comment:
The removal of extra brackets`(` `)` in the link should fix the current build error. 
_Refer below suggestion._ 


```suggestion
- `operator`は、`<`(より小さい)などの[関係演算子](https://ja.wikipedia.org/wiki/%E9%96%A2%E4%BF%82%E6%BC%94%E7%AE%97%E5%AD%90#%E6%A8%99%E6%BA%96%E7%9A%84%E3%81%AA%E9%96%A2%E4%BF%82%E6%BC%94%E7%AE%97%E5%AD%90)です。
```